src/css/styles.scss

```diff
@@ -549,6 +549,7 @@
     border-top-left-radius: var(--border-radius-lg);
     border-bottom-left-radius: var(--border-radius-lg);
     border-right: 0;
+    overflow: hidden;
```

Comment:
for what is this needed ?

Reply:
![image](https://user-images.githubusercontent.com/5153846/214054454-abde886d-8a00-4725-86d7-06c11ee862ce.png)